### DISCUSSION TITLE
priority of operators in math.parse()

### DISCUSSION THREAD

**Discussion by meghna786** [josdejong/mathjs#3108] [Q&A] [answered]
if I type math.parse('a*b/c')
then multiplication is done first

I want it to divide first as according to DMAS, divide has more priority! Is there any way I can change priorities of operators for math.parse?

**Comment by josdejong** [comment, marked as answer]
The precedence of multiplication and division in mathjs is hardcoded. If you want to change that, you can either:

- clone the library and change the code of the parser, [see parse.js](https://github.com/josdejong/mathjs/blob/develop/src/expression/parse.js#L1031-L1060).
- write logic to adjust the parsed expression tree using the `Node.transform` method. [See docs](https://mathjs.org/docs/expressions/expression_trees.html) (search for "transform" on that page).

**Reply by meghna786** [reply]
thanks, I'll look into it.
